test/functional/wallet_fundrawtransaction.py

```diff
@@ -1064,8 +1064,7 @@ def test_external_inputs(self):
 
         # Funding should also work if the input weight is provided
         funded_tx = wallet.fundrawtransaction(raw_tx, input_weights=[{"txid": ext_utxo["txid"], "vout": ext_utxo["vout"], "weight": input_weight}])
-        signed_tx = wallet.signrawtransactionwithwallet(funded_tx["hex"])
```

Comment:
I might be misunderstanding, but should we be keeping the `wallet.signrawtransactionwithwallet` instead, since `ext_utxo` is external to node2 and not node0?